news/2019-10-12-beatmap-pack-management-update-#01.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Beatmap Pack Management: Update #1"
```

Comment:
this title doesn't really suggest that there are new packs, it sounds like a general update about your project

I think "New Beatmap Packs Available!" would be enough ?

Reply:
Since the post covers new packs but also informations about ongoing things  I think an 'update' sounds more appropriate.

Reply:
still doesn't feel like a good hook considering this is the first mention of Beatmap Pack Management for most of your readers. also, it doesn't have to be so structured/repeatable (in case you thought it did, idk)

- Breathing New Life into Beatmap Packs with Beatmap Pack Management
- New Beatmap Packs and New Beatmap Packers
- Introducing Beatmap Pack Management

for examples

Reply:
chose the last example.